### PR TITLE
phase1 docs add

### PR DESCRIPTION
**Pull request by rochelle8044**

### CODE REVIEW

File: docs/stake.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Stake AGE has several benefits over regular staking:
+
+- Users will be incentivized to keep their tokens staked for a longer time.
+- Early and long-term participants are rewarded.
+- It will be harder for new members with a lot of tokens to immediately take big rewards or votes from the DAO.
+- The penalty for unstaking will be bigger as users will lose their Stake AGE for their unstaked tokens, making sure users are incentivized to keep their tokens staked.
+
+## EFX Power
+
+EFX Power is a factor that determines your weight and rank within the DAO. This measurement is a result of the Staking Protocol 2.0. There are two ways to increase your EFX Staking Power:
```
Comment (djmbritt) [inline docs/stake.md]: Might be nice to mention the proposal that led to staking protocol 2.0 here.

File: docs/stake.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+and the *unstake delay time* is started. After this delay the tokens will be
+*refunded* to your wallet.
+
+When you unstake EFX tokens you also loose their stake age. Of course the
+remaining staked tokens will keep their age but if you were to re-stake the
+unstaked portion they would dilute your age.
+
+## Refunding
+
+Usually your unstaked tokens are automatically transferred back to your wallet
```
Comment (djmbritt) [inline docs/stake.md]: **"Usually"** is a bit inconspicuous / vague here. Especially when it's talking about unstaking tokens. The statement should be absolute about what is supposed to happen here.

File: docs/blockchainwallets.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Blockchain Wallets
+
+# What is a crypto wallet?
+
+In essence, a cryptocurrency wallet is a gadget that allows you to communicate with a blockchain network. There are various types of cryptocurrency wallets, which can be divided into three categories: software, hardware, and paper wallets. Depending on how they work, they may alternatively be referred to as "hot" or "cold" wallets.
```
Comment (djmbritt) [inline docs/blockchainwallets.md]: The word gadget feels weird here. I've never heard of an application called as a gadget.

File: docs/blockchainwallets.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+The wallet also contains an address, which is an alphanumeric identifier formed from the public and private keys. In essence, an address is a distinct "location" on the blockchain to which bitcoin can be transmitted. This implies that while you can share your address with others in order to receive payments, you should never reveal your private key with anyone.
+
+Regardless of whose wallet you use, the private key grants access to your cryptocurrency. As a result, even if your computer or smartphone is compromised, you can still access your assets on another device if you have the associated private key (or seed phrase). It is important to note that the coins are never completely removed from the blockchain; they are simply transferred from one address to another.
+
+# Types of wallets
+
+## Hardware wallets
```
Comment (djmbritt) [inline docs/blockchainwallets.md]: Might be useful to add a couple of links where users could buy a hardware wallet?
Is it a good/bad idea to add affiliate links?

File: docs/blockchainwallets.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+Because of this, mobile wallets are the best way to make daily payments and transactions. This makes them a good way to spend Bitcoin, BNB, and other cryptocurrencies in the real world. One well-known example of a mobile crypto wallet is Trust Wallet.
+
+Like desktops, mobile devices can get messed up by rogue programs and malware. So, you should protect your mobile wallet with a password and make a copy of your private keys (or seed phrase) in case you lose or damage your phone.
+
+# The importance of backups
+
+It might be expensive to lose access to your cryptocurrency wallet. As a result, regular backups are essential. This is frequently accomplished by simply backing up wallet.dat files or seed phrases. A seed phrase, in essence, functions as a root key, generating and granting access to all keys and addresses in a crypto wallet. Also, if you chose password encryption, keep a backup of your password.
+
+# Supported wallets for the Effect Network
```
Comment (djmbritt) [inline docs/blockchainwallets.md]: I think this paragraph should be at the top, it's the most important bit for our users.

File: docs/efxtoken.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# The $EFX Utility Token
```
Comment (djmbritt) [inline docs/efxtoken.md]: This file should also contain information about the farm.

**Comment by laurensV** [comment]
@rochelle8044 you had another PR https://github.com/effectai/effect-network/pull/100 that was merged that is causing a merge conflict. Could you resolve the merge conflict so that we can also merge this? Let me know if you need help with that

**Pull request merged**